src/components/mobile/mobile-next-query-preview.vue

```diff
@@ -48,7 +48,7 @@
       :showButtons="false"
```

Comment:
You mean the `padding-left: var(--x-size-padding-grid);` ? I don't think we can remove that, the XDS sliding panel doesn't care about that padding.

Reply:
Oh, I see that we are still using the `x-sliding-panel__scroll` in the component.
Nevermind, I would add that padding directly using the `scrollContainerClass="x-pl-16"` instead of with an internal selector.

The padding plugin needs to be enabled.

Reply:
Well, actually we can use `x-px-16` to apply it to the right too.